ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/431__data-science-pipelines-api.robot

```diff
@@ -40,6 +40,21 @@ Verify Ods Users Can Do Http Request That Must Be Redirected to Https
     Should Start With    ${url}    https
     Remove Pipeline Project    project-redirect-http
 
+Verify DSPO Operator Reconciliation Retry
+    [Documentation]    Verify DSPO Operator is able to recover from missing components during the initialization
+    [Tags]      Sanity    Tier1    TODO
+    ${local_project_name} =    Set Variable    recon-test
+    New Project    ${local_project_name}
+    Install DataSciencePipelinesApplication CR    ${local_project_name}    data-science-pipelines-reconciliation.yaml    False
+    Wait Until Keyword Succeeds    15 times    1s
+    ...    Double Check If DSPA Was Created    ${local_project_name}
```

Comment:
Could be refactored into a couple of new keywords in ODHDataSciencePipelines.resource:

- `Wait Until DataSciencePipelineApplication Exists Using CLI  $project $dspa_name`
- `Verify DataSciencePipelineApplication Exists Using CLI $project $dspa_name`

Reply:
The same from above. Fell free to rename it on a new PR